subsys/bluetooth/mesh/main.c

```diff
@@ -51,13 +51,15 @@ int bt_mesh_provision(const uint8_t net_key[16], uint16_t net_idx,
 	struct bt_mesh_key mesh_net_key;
 	bool is_net_key_valid = false;
 	bool is_dev_key_valid = false;
+	bool is_cdb_dev_key_valid = false;
 	int err = 0;
 
 	if (!atomic_test_bit(bt_mesh.flags, BT_MESH_INIT)) {
 		return -ENODEV;
 	}
 
 	struct bt_mesh_cdb_subnet *subnet = NULL;
+	struct bt_mesh_cdb_node *node = NULL;
```

Comment:
Does it need to be initialized (as well as `subnet`)?

Reply:
actually not, just leftover from one of the intermediate implementation variants. I'll remove it.